app/resources/v1/auth.py

```diff
@@ -0,0 +1,86 @@
+
+import pdb
```

Comment:
Why is this here

Reply:
Try and resolve merge conflicts too

Reply:
I was debugging and left the pdb lying there. Thanks for pointing that out